### PR TITLE
Clean code with TSLint

### PR DESCRIPTION
**Pull request by stelar7**

### CODE REVIEW

File: src/ApiStatus.ts [hunk 1]
```diff
@@ -43,70 +43,69 @@ export default class ApiStatus {
...
 
-    onBot() {
+    private onBot() {
         console.log("API Status extension loaded.");
     }
 
-    async onInfo(message: Discord.Message) {
-        if (message.author.bot) return;
+    private async onInfo(message: Discord.Message) {
+        if (message.author.bot) { return; }
```
Comment (molenzwiebel) [inline src/ApiStatus.ts]: ew

File: src/ApiStatus.ts [hunk 2]
```diff
@@ -43,70 +43,69 @@ export default class ApiStatus {
...
-        if (!this.isApiStatusCommand(content))
-            return;
+        if (!this.isApiStatusCommand(content)) { return; }
 
         const apiStatus = await this.getApiStatus();
 
-        const fields: {name: string, value: string, inline: boolean}[] = [];
-        for (const api in apiStatus.api){ 
-            fields.push({name: api, value: apiStatus.api[api], inline: true});
+        const fields: Array<{ name: string, value: string, inline: boolean }> = [];
```
Comment (molenzwiebel) [inline src/ApiStatus.ts]: ew

File: src/ChannelAccess.ts [hunk 3]
```diff
@@ -82,9 +83,9 @@ export default class ChannelAccess {
         }
 
         if (action === "join") {
-            this.joinChannel(message, <Discord.TextChannel>channel);
+            this.joinChannel(message, channel as Discord.TextChannel);
         } else if (action === "leave") {
-            this.leaveChannel(message, <Discord.TextChannel>channel);
+            this.leaveChannel(message, channel as Discord.TextChannel);
```
Comment (molenzwiebel) [inline src/ChannelAccess.ts]: ¯\_(ツ)_/¯

File: src/ApiStatus.ts (hunk 1 above)
Comment (Querijn) [reply src/ApiStatus.ts]: agreed

File: src/ChannelAccess.ts (hunk 3 above)
Comment (Querijn) [reply src/ChannelAccess.ts]: you dropped this \

**Comment by Querijn** [comment]
X[] is my preference as well

File: src/ChannelAccess.ts (hunk 3 above)
Comment (stelar7) [reply src/ChannelAccess.ts]: no-angle-bracket-type-assertion is default because of tsx, should we keep <> or stick to `as`?

**Comment by Querijn** [comment]
Assuming no change in functionality

**Pull request merged**